README.md

```diff
@@ -86,8 +86,67 @@ Usage of ./hetty:
         CA private key filepath. Creates a new CA private key if file doesn't exist (default "~/.hetty/hetty_key.pem")
 ```
 
-⚠️ _Todo: Write instructions for installing CA certificate in local CA store, and_
-_configuring Hetty to be used as a proxy server._
+## Certificate Setup and Installation
+
+In order for Hetty to proxy requests going to HTTPS endpoints, certificates for 
+Hetty will need to be set up. Furthermore, these certificates may need to be
+installed to the host for them to be trusted by your browser. The following steps
+will cover how you can generate your certificate, provide them to hetty, and how
+you can install them in your local CA store.
+
+⚠️ _Note: These instructions assume you have access to the OpenSSL libraries and_
+_command line utilities. This process was done on a Linux machine but should_
+_provide guidance on Windows and macOS as well._
+
+### Generating CA certificates
+
+You can start off by generating a new key and CA certificate which will both expire
+after a month.
+
+```sh
+openssl req -newkey rsa:2048 -new -nodes -x509 -days 31 -keyout hetty_key.pem -out hetty_cert.pem
```

Comment:
I don't think that's needed anymore.
Hetty generates a set of certs on startup

Reply:
Good point! I can make a mention of the automatically generated certificates as well 👍

Reply:
Latest commit 9a8afda now mentions instructions for both Hetty and OpenSSL, mentioning the benefit of one over the other, and suggesting Hetty defaults for beginners. More advanced users who need control over certificate expiration or cryptography strength can use OpenSSL if they would like, and the instructions outline the process.